src/routes/docs/components/Explore.svelte

```diff
@@ -53,8 +50,8 @@
   </div>
 
   <!-- min width selected based on description section + 1 open accordion + 3 closed accordions -->
-  <div class="w-full bg-navy sm:px-f32 sm:py-f16">
-    <div class="flex flex-col gap-f4 lg:min-w-[716px] lg:flex-row vertical-md:flex-row">
+  <div class="w-full sm:px-f32 sm:py-f16">
+    <div class="grid grid-cols-1 gap-x-f4 gap-y-f48 md:grid-cols-2 lg:flex">
```

Comment:
Swap to grids.

I tried doing the grid for the open/close, but in the end it was ugly with tailwind, so stuck with the flex for the lg